assets/src/edit-story/components/helpCenter/quickTip/index.js

```diff
@@ -73,8 +73,14 @@ const DoneContainer = styled.div`
   ${themeHelpers.centerContent}
   height: 180px;
   margin-bottom: ${GUTTER_WIDTH}px;
+  color: #f4f2ef;
+
   svg {
     display: block;
+
+    path {
+      color: #4285f4;
+    }
```

Comment:
Should these come from the theme?

Reply:
These colors don't exist anywhere else except for this svg 😞. I don't think we should add these to the design system until they're needed more than once.

This is why I wanted these colors to be applied to the svg without styled components since the colors are strange and don't belong in the design system 🤔 Happy to do either way though.

Reply:
Worth clarifying with UX then to see whether this is expected or if this is an oversight and the color should be changed to something that already exists.

Reply:
At least `#4285f4` is/was used/defined in `edit-story` in a few places, so perhaps it needs to be replaced with something newer.